tests-upgrade/AutoRestUpgradeTest.ps1

```diff
@@ -105,14 +105,12 @@ function CompareGeneratedCode([string]$inputSourcePath,[string]$inputTargetPath,
         $ignoreResult = IsNeedIgnore -inputFileName $initFile.FullName -ignoreArray $initIgnoreFileList
         if(!$ignoreResult)
         {
-            # if(!$initFile.FullName.Startswith($initIgnoreFileList)){
             $obj = "what" | Select-Object -Property HashCode, Status
             #if the file is not filefolder
-            if($initFile.mode -eq '-a---')
+            if($initFile.mode.Startswith('-a'))
```

Comment:
Please use file method instead of comparing mode string directly.

Reply:
Please use more stable way to check, here is a example:
```powershell
(Get-Item .\src\Resources\Resources\ChangeLog.md) -is [System.IO.DirectoryInfo]
```

Reply:
Done，thanks